src/cplscheme/config/CouplingSchemeConfiguration.hpp

```diff
@@ -156,7 +155,8 @@ class CouplingSchemeConfiguration : public xml::XMLTag::Listener {
     };
     std::vector<Exchange>                    exchanges;
     std::vector<ConvergenceMeasureDefintion> convergenceMeasureDefinitions;
-    int                                      maxIterations = -1;
+    int                                      maxIterations = -1; // infinite by default
```

Comment:
See above. I think using a `const int INFINITE_MAX_ITERATIONS = -2` would be good here. Currently also overlaps with `UNDEFINED_MAX_ITERATIONS`.

Reply:
Done, I used `UNDEFINDED_MXX_ITERATIONS` for both. For implicit cases, I convert `UNDEFINED_MAX_ITERATIONS` to `INFINITE_MAX_ITERATIONS`